tests/fw_rules_tests/policies/expected_output/calico-testcase20-scheme_output.csv

```diff
@@ -1,70 +1,70 @@
 "query","src_ns","src_pods","dst_ns","dst_pods","connection",
 "Eran_gnps, config: Eran_gnps","","","","","",
-"","","ip block: 0.0.0.0/0","[None]","[vendor.role in (worker_public)]","ICMP [0-254] => [0-255];,TCP 52311,UDP 52311,protocols numbers: 112",
-"","[None]","[vendor.role in (worker_public)]","[None]","[vendor.role in (worker_public)]","ICMP [0-254] => [0-255];,TCP 52311,UDP 52311,protocols numbers: 112",
-"","[kube-system]","[*]","[None]","[vendor.role in (worker_public)]","ICMP [0-254] => [0-255];,TCP 52311,UDP 52311,protocols numbers: 112",
+"","","ip block: 0.0.0.0/0","[None]","[has(vendor.role)]","ICMP [0-254] => [0-255];,TCP 52311,UDP 52311,protocols numbers: 112",
```

Comment:
I'm not sure I understand how this change happened. Is it correct?

Reply:
I've removed the outputConfig option `fwRulesGeneralizeLabelExpr `and applied it to all fw-rules, so in this case the label-expr of `"vendor.role in (worker.public)"` was generalized to `"has(vendor.role)"`, and it is correct for this test's topology.